### PR TITLE
Added metrics for ru convergence time

### PR DESCRIPTION
**Pull request by akun1012**

### CODE REVIEW

**Comment by esuen** [comment]
This is open src and public, so you probably don't want to reference the internal ticket.

File: pkg/roll/fields/update.go [hunk 1]
```diff
@@ -27,8 +27,8 @@ type Update struct {
 	//
 	// The acceptable value of Minimum is subject to several constraints:
 	//   - the update will not even start until the minimum is satisfied, so if
-	//     the current number of live nodes is than the minimum, it will block
-	//     forever.
+	//     the current number of live nodes is smaller than the minimum, it will
```
Comment (esuen) [inline pkg/roll/fields/update.go]: nit: "less" instead of "smaller"

**Comment by akun1012** [comment]
@esuen Thanks.

This PR could is with build failure now. Will modify, and notify when it's ready to review.

**Comment by akun1012** [comment]
Build passed. Please review.

**Comment by rudle** [comment]
Looks good! Please squash these commits into one before merging. One note: imports should always be added with the code that references them, please configure your editor to organize imports on file save. You might need to install `goimports`.

**Pull request merged**